compiler/onecc-docker/onecc-docker

```diff
@@ -44,14 +47,47 @@ def _image_exists(name):
 
 def main():
     # TODO Update Dockerfile path
-    dockerfile_path = "."
+    dockerfile_path = "./docker"
+
+    onecc_docker_usage = 'onecc-docker [-h] [-t TOKEN] [--] [COMMANDS FOR ONECC]'
+    onecc_docker_desc = 'Run onecc via docker'
+    parser = argparse.ArgumentParser(
+        usage=onecc_docker_usage, description=onecc_docker_desc)
+    parser.add_argument(
+        "-t",
+        "--token",
+        help=
+        "Token for authentication to GitHub. This is a workaround for Rate limit exceeded error"
+    )
+
+    onecc_docker_args = []
+    onecc_args = []
+    argv = copy.deepcopy(sys.argv)
+    # delete file name
+    del argv[0]
+    # split by '--'
+    args = [list(y) for x, y in itertools.groupby(argv, lambda z: z == '--')]
+    # onecc-docker has two interfaces
+    # 1. onecc-docker [-h] [-t TOKEN] [COMMANDS FOR ONECC]
```

Comment:
This condition should be removed. https://github.com/Samsung/ONE/pull/9800#discussion_r980900351 was just for example. This was for backward compatibility. Since `onecc-docker` is just about to introduced, this condition doesn't need to be added.